earthpy/tests/test_stack_raster_tifs.py

```diff
@@ -0,0 +1,29 @@
+""" Tests for the _stack_bands() method """
+
+import numpy as np
+import numpy.ma as ma
+import pandas as pd
+import pytest
+import geopandas as gpd
+import rasterio as rio
+from shapely.geometry import Polygon, Point, LineString
+import earthpy.spatial as es
+import os
+
+
+@pytest.fixture
+def b1_b2_arrs():
+    b1 = np.array([[6, 7, 8, 9, 10], [16, 17, 18, 19, 20]])
+    b2 = np.array([[1, 2, 3, 4, 5], [14, 12, 13, 14, 17]])
+    return b1, b2
+
+
+def test_stack_raster_tifs(b1_b2_arrs):
+    """Unit test for stack_raster_tifs()."""
+
+    # Test data
+    b1, b2 = b1_b2_arrs
+
+    # Check Warning for Deprecation
+    with pytest.raises(Warning):
```

Comment:
@joemcglinchy  i think you want `(Warning, match="Beginning of the expected warning raised here")` to ensure the proper message is raised with the warning

Reply:
I think also you can also remove both comments in this test!

Reply:
the pytest call did not evaluate correctly when checking for the match string, so i removed it.

Reply:
ok! i gave it a go locally and it seemed to work... i added a commit and let's see if travis yells at us or not! i've found that i don't use the ENTIRE message with match="" just the first part as spaces can throw it off. anyway let's see what happens. if it fails ill undo it!!